pages/workspace/workspace.ejs

```diff
@@ -1,30 +1,72 @@
 <html>
-<head>
+  <head>
     <%- include('../partials/head'); %>
-</head>
-<body style="margin:0; overflow: hidden;" width="100%" height="100%">
-    <iframe id="workspace_container" width="100%" height="100%" style="margin: 0; border: none;"></iframe>
-    <button id="button" style="position: absolute; left: 250px; bottom: 0">Ping the Server</button>
+  </head>
+  <body style="margin: 0; overflow: hidden; height: 100%; width: 100%;">
+    <div>
+      <%- include('../partials/navbar', {navPage: ''}); %>
+      <style>
+        .mb-4 {
+          margin-bottom: 0 !important;
+        }
+      </style>
+    </div>
+    <div style="position: fixed; height: 100%; width: 100%;">
+      <div id="loading_page"><%- include('../partials/loading'); %></div>
+      <iframe
+        id="workspace_container"
+        style="margin: 0; border: none; width: 100%; height: calc(100% - 57px);"
+      ></iframe>
+    </div>
+    <button id="button" style="position: absolute; left: 250px; bottom: 0;">
+      Ping the Server
+    </button>
     <script src="/javascripts/socket.io.js"></script>
```

Comment:
```suggestion
    <script src="/node_modules/socket.io-client/dist/socket.io.js"></script>
```

Local copy of `socket.io` was removed in https://github.com/PrairieLearn/PrairieLearn/commit/76f580ce55000ea0e13531735e00bc5d359ab4e3 in favor of the copy in node_modules.